docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc

```diff
@@ -0,0 +1,55 @@
+= Releasing with an advisory
+
+In order to be able to create an advisory when releasing a Snapshot, certain data has to be included in the Release Plan Admission.
+
+.*Prerequisites*
+
+* You have an existing Release Plan Admission for your release.
+
+.*Procedures*
+
+. Add the advisory data in your ReleasePlanAdmission object.
+
+*Example `ReleasePlanAdmission.yaml` object*
+
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  ..
+spec:
+ data:
+   releaseNotes: <advisory data> <.>
+----
+<.> Advisory data to be included.
+
+.*UMB messages*
+
+When an advisory yaml is created and pushed to the advisories Gitlab repo, a message is sent automatically to UMB (Universal Message Bus) via Gitlab CI.
```

Comment:
Also, this doesn't make sense in the upstream docs. We can leave it here for now, but we should move this out to a downstream-only doc once we have a process established to integrate those.

Reply:
Should we just move the entire doc to a downstream doc? I am not sure it makes sense to have any of it upstream. Maybe the advisory repo README?

Reply:
I agree the whole doc would probably make sense somewhere internally, but not sure where.